### ISSUE TITLE
Which file and class is the L2 norm attack?

### ISSUE THREAD

**Issue by machanic** [cmhcbb/attackbox#5]
How to support L2 norm attack? I can only find `Sign_OPT_lf.py` as the Linf norm attack? 

Is `Sign_OPT_lf.py` and `class OPT_attack_sign_SGD` the L2 norm attack??

**Comment by cmhcbb** [comment]
Yes, Sign_OPT.py is the l2 version. You could check test_attack.py for the example to run the attack.

**Comment by machanic** [comment]
@cmhcbb What is `Sign_OPT_v2.py` used for? It is not the file of L2 norm and L-inf norm attack? Then what is it?

**Comment by cmhcbb** [comment]
It's a file for debugging purposes.

**Comment by machanic** [comment]
@cmhcbb Dear Sir, where is the code of targeted attack of L2 norm? I post this issue to https://github.com/cmhcbb/attackbox/issues/6
